src/index.ts

```diff
@@ -0,0 +1,22 @@
+export class Challenge
+{
+    public buzz(n: number) :Promise<any> 
```

Comment:
`any` is for very special cases, you should avoid it as much as possible. In this case the Promise will resolve to a `string`  or a `number`. You can express this with a [union type](https://www.typescriptlang.org/docs/handbook/unions-and-intersections.html):
```suggestion
    public buzz(n: number): Promise<string | number> 
```